src/resources/wifi_esp32.cc

```diff
@@ -337,11 +337,25 @@ PRIMITIVE(disconnect_reason) {
   }
 }
 
+static char local_address[17] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
I think this should be 16 and below the memcpy should be 15.

Reply:
done.